### PR TITLE
Fix lint GitHub workflow

### PR DESCRIPTION
**Pull request by johnnyomair**
The lint GitHub workflow had multiple issues:
1. The `lint:ci` scripts  weren't used
2. The lint for generated files in the Admin was calling the API Generator 😂 
3. The Admin Generator wasn't working due to missing the config file

### CODE REVIEW

File: admin/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         "@comet/admin-rte": "^7.8.0",
         "@comet/admin-theme": "^7.8.0",
         "@comet/blocks-admin": "^7.8.0",
-        "@comet/cms-admin": "^7.8.0",
+        "@comet/cms-admin": "^7.9.0-canary-20241203145831",
```
Comment (johnnyomair) [inline admin/package.json:35]: Need to wait for a stable release that includes https://github.com/vivid-planet/comet/pull/2870.

**Review by dkarnutsch — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -10,6 +10,11 @@
         "lint:api": "npm --prefix api run lint",
         "lint:admin": "npm --prefix admin run lint",
         "lint:site": "npm --prefix site run lint",
+        "lint:ci": "run-p lint:ci:*",
+        "lint:ci:root": "npm run lint:root",
+        "lint:ci:api": "npm --prefix api run lint:ci",
+        "lint:ci:admin": "npm --prefix admin run lint:ci",
+        "lint:ci:site": "npm --prefix site run lint",
```
[the remark this answers is not in the document]
Comment (johnnyomair) [reply package.json]: I don't like adding things that will never be used in the projects. I've changed the lint step to use the lint:ci scripts from the micorservices: 818feb23dafd991b59030c7f1f71d66ac27b4623.

**Review by dkarnutsch — DISMISSED** [review]

File: site/package.json [hunk 3]
```diff
@@ -12,6 +12,7 @@
         "intl:extract": "formatjs extract \"src/**/*.ts*\" --ignore **/*.d.ts --out-file lang-extracted/en.json --format simple --throws",
         "prelint": "npm run intl:compile && run-p gql:types generate-block-types",
         "lint": "run-p lint:eslint lint:tsc lint:style intl:extract",
+        "lintci": "npm run lint",
```
Comment (dkarnutsch) [inline site/package.json:15]:
```suggestion
        "lint:ci": "npm run lint",
```

**Review by dkarnutsch — DISMISSED** [review]

File: site/package.json (hunk 3 above)
Comment (johnnyomair) [reply site/package.json]: 🤦🏼‍♂️

**Review by dkarnutsch — APPROVED** [review]

**Pull request merged**